Foundation/NSString.swift

```diff
@@ -1292,9 +1297,11 @@ extension NSString {
         // manually skip the BOM header.
         guard let cf = CFStringCreateWithBytes(kCFAllocatorDefault, bytePtr + offset, readResult.length - offset,
                                                CFStringConvertNSStringEncodingToEncoding(encoding), true) else {
-            throw NSError(domain: NSCocoaErrorDomain, code: CocoaError.fileReadInapplicableStringEncoding.rawValue, userInfo: [
-                "NSDebugDescription" : "Unable to create a string using the specified encoding."
-                ])
+                                                throw NSError(domain: NSCocoaErrorDomain,
```

Comment:
Can you revert this reformatting, since it isn't really relevant or part of this change?